src/index.js

```diff
@@ -1,13 +1,42 @@
-import _ from 'lodash';
+/* eslint-disable no-return-assign */
 import './style.css';
 
-function component() {
-  const element = document.createElement('div');
+const url = 'https://us-central1-js-capstone-backend.cloudfunctions.net/api/games/mchCFxyMLOw3tEZP2NKf/scores/';
+const leaderboard = document.getElementById('leaderboard');
+const refresh = document.getElementById('refresh');
+const name = document.getElementById('name');
+const score = document.getElementById('score');
+const submit = document.getElementById('add');
 
-  // Lodash, imported
-  element.innerHTML = _.join(['Hello', 'webpack'], ' ');
+const createGame = (name) => {
+  leaderboard.innerHTML = '';
+  name.result.map((e, index) => leaderboard.innerHTML += `<li class="list-group-item">${name.result[index].user} ${name.result[index].score}</li>`);
 
-  return element;
-}
+  leaderboard.appendChild(name);
+};
 
-document.body.appendChild(component());
\ No newline at end of file
+const callApi = async () => {
+  const getData = await fetch(url);
+  const resource = await getData.json();
+  createGame(resource);
+  return resource;
+};
+
+callApi();
+
+const addScore = () => {
```

Comment:
* I get this response (status 400) when i try to add a score kindly fix it 🙏 
![image](https://user-images.githubusercontent.com/52497246/128316177-9c9879eb-99ec-48b5-96d4-5410a7bd6978.png)

Reply:
Well addressed this!